src/mpl/include/mpl_trmem.h

```diff
@@ -113,6 +113,78 @@ char *MPL_strdup(const char *str);
 #endif /* MPL_USE_MEMORY_TRACING */
 
 
+/* --------------------------------------------------------------------
+ * MPL memory alignment union
+ * The MPL_mem_alignment_t union is used to help internal structures or buffers
+ * follow the alignment rules for all predefined datatypes. */
+
+#if 0 /* sample usage : wrap up sample_t structure. */
+typedef union {
+    sample_t var;
+    MPL_mem_alignment_t alignment;
+} aligned_sample_t;
+#endif
+
+typedef struct {
+    double re;
+    double im;
+} mpl_d_complex;
+
```

Comment:
You don't need a separate named structure here. You can simply declare the structure inside the union you are using it in.